nilearn/signal.py

```diff
@@ -275,46 +275,46 @@ def butterworth(signals, sampling_rate, low_pass=None, high_pass=None,
 def high_variance_confounds(series, n_confounds=5, percentile=2.,
                             detrend=True):
     """ Return confounds time series extracted from series with highest
-        variance.
-
-        Parameters
-        ==========
```

Comment:
There are remaining `=` in other places in your diff too, that'd be great to fix them while you are at it.